ogr/services/pagure/project.py

```diff
@@ -208,8 +208,10 @@ def get_issue_list(
     def get_issue(self, issue_id: int) -> Issue:
         return PagureIssue.get(project=self, id=issue_id)
 
-    def create_issue(self, title: str, body: str) -> Issue:
-        return PagureIssue.create(project=self, title=title, body=body)
+    def create_issue(
+        self, title: str, body: str, tags: Optional[List[str]] = None
```

Comment:
@lachmanfrantisek would you also change `tags` here for `labels` so that it is consistent ?

Reply:
Yes, that would be nice.